src/feature/projects/components/Projects.tsx

```diff
@@ -5,7 +5,7 @@ import { cleanStringFromHtlmTags, rowalizer } from "../../../utils/helpers"
 import SeoLink from "../../../components/shared/SeoLink"
 import { MobileOnlyProjectCard } from "./ProjectCard"
 import { ProjectContent } from "./ProjectContent"
-import { ItemImage } from "../../guide/GuideItem"
+import { ItemImage } from "../../../components/shared/ItemImage"
```

Comment:
@leonardodemarco00 Dai changelog non lo vedo ma se non usassimo più `ProjectImage` uccidi pure il file

Reply:
Era usato ancora in ArticleFooter, ma l'ho sostituito così abbiamo un pisciaturo un meno